stdlib/public/core/StringUTF8View.swift

```diff
@@ -317,6 +317,40 @@ extension String.UTF8View {
   }
 }
 
+extension String.UTF8View {
+
+  @available(SwiftStdlib 6.2, *)
+  public var span: Span<UTF8.CodeUnit> {
+    @lifetime(borrow self)
```

Comment:
```suggestion
    @lifetime(borrow self)
    @_alwaysEmitIntoClient
```

Reply:
I'm not convinced we want these to be inlinable.

Reply:
None of the other `span` properties are ABI, so I thought these shouldn't be either.
Would a combination of `@_alwaysEmitIntoClient` and `@inline(never)` work?

Reply:
The biggest advantage to making the others inlinable is to avoid trampolining through generic code to get to a concrete `Span`. We could probably make them `@inlinable` (and ABI) rather than `@_alwaysEmitIntoClient`, but there isn't much advantage there. In the `String` case we don't have the generics concern, but there is a large amount of machinery protected by `internal`; making this a non-inlinable symbol ensures there are no accidentally-exposed internal implementation details.

Reply:
We need to be extraordinarily careful not leak implementation details about ObjC bridging into client binaries -- we want to keep these entirely in the stdlib, so that it can be freely changed later. I think this implementation _must_ remain opaque.

If we want to back-deploy this entry point, then we can do that by have it dispatch to the opaque implementation on future OS versions only, with only the backward-deploying variant being marked `@_aeic`. That assumes it is _possible_ to back-deploy this property in the first place -- that would need to be carefully verified.